docs/source/deployment/heroku.md

```diff
@@ -4,6 +4,8 @@ sidebar_title: Heroku
 description: Deploying your GraphQL server to Heroku
 ---
 
+FIXME AS3: Walk through and test this whole file.
```

Comment:
This one has an open communityPR that I'm going to address shortly, and I can definitely tackle this one